hypervisor/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2018"
 num_enum = { version="0.5.1", default-features=false }
 log = { version="0.4.11", default-features=false }
 elf_rs = { version="0.1.3", default-features=false }
-rustsbi = "0.2.1"
+rustsbi = { version = "0.4.0-alpha.1", git = "https://github.com/rustsbi/rustsbi", features = ["forward"]}
```

Comment:
I'll send another pull request once stable RustSBI 0.4.0 is formally released :)